R/Rules-methods.R

```diff
@@ -1725,6 +1730,15 @@ setMethod("stopTrial",
       ## so can we stop?
       doStop <- nCohorts >= stopping@nCohorts
 
+      report_label <- if (isTRUE(stopping@report_label == "")) {
+        paste(
+          "\u2265", x@nCohorts, "cohorts in", x@percentage,
```

Comment:
what is the `x` here? I do not see this object on the list of explicit arguments to this function.